app-shells/rc/rc-1.7.4-r2.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="A reimplementation of the Plan 9 shell"
+HOMEPAGE="http://static.tobold.org/"
+SRC_URI="http://static.tobold.org/${PN}/${P}.tar.gz"
+
+LICENSE="rc"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="libedit readline"
+
+RDEPEND="readline? ( sys-libs/readline:0 )
+	libedit? ( dev-libs/libedit )"
+DEPEND="${RDEPEND}"
+
+DOCS=( AUTHORS ChangeLog NEWS README )
+
+src_configure() {
+	local myconf="--with-history"
+	use readline && myconf="--with-edit=readline"
+	use libedit && myconf="--with-edit=edit"
+
+	econf "${myconf}"
+}
+
+src_install() {
+	into /usr
+	newbin "${PN}" "${PN}sh"
```

Comment:
I would personally not bother quoting ${PN} as it's guaranteed to never contain a space.